### PR TITLE
[DOC] Updated changelog for 1.0 release

### PR DESCRIPTION
**Pull request by yunchu**
Updated CHANGELOG.md for the 1.0.0 release.

(include back-merge of changelog from v0.5 release)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,48 @@
 
 All notable changes to this project will be documented in this file.
 
+> _**notes**_
+> OpenVINO Training Extension which version is equal or older then v0.5.0 does not include the latest functional and security updates. OTE Version 1.0.0 is targeted to be released in February 2023 and will include additional functional and security updates. Customers should update to the latest version as it becomes available.
+
+## \[v1.0.0\]
+
+### Feature
```
Comment (goodsong81) [inline CHANGELOG.md]: How about "New Features"?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,48 @@
 
 All notable changes to this project will be documented in this file.
 
+> _**notes**_
+> OpenVINO Training Extension which version is equal or older then v0.5.0 does not include the latest functional and security updates. OTE Version 1.0.0 is targeted to be released in February 2023 and will include additional functional and security updates. Customers should update to the latest version as it becomes available.
```
Comment (goodsong81) [inline CHANGELOG.md]:
```suggestion
> OpenVINO Training Extension v0.5.0 and older version do not include the latest features and security updates. Please upgrade the package once v1.0.0 is released in February 2023.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,48 @@
...
+
+### Feature
+
+- Adaptation of [Datumaro](https://github.com/openvinotoolkit/datumaro) component as a dataset interface
+- Integrate hyper-parameter optimizations
+- Support action recognition task
+- Self-supervised learning mode for representational training
+- Semi-supervised learning mode for better model quality
+
+### Enhancement
```
Comment (goodsong81) [inline CHANGELOG.md]:
```suggestion
### Enhancements
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,48 @@
...
+
+- Installation through PyPI - Named as [OTX](https://pypi.org/project/otx/)
```
Comment (goodsong81) [inline CHANGELOG.md]:
```suggestion
- Installation via [PyPI package](https://pypi.org/project/otx/)
```
Not to user OTX term.

**Review by goodsong81 — CHANGES_REQUESTED** [review]
Thank you for the update!
I've requested a few changes.

**Review by chuneuny-emily — DISMISSED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,53 @@
...
+- Adaptation of [Datumaro](https://github.com/openvinotoolkit/datumaro) component as a dataset interface
+- Integrate hyper-parameter optimizations
+- Support action recognition task
+- Self-supervised learning mode for representational training
+- Semi-supervised learning mode for better model quality
+
+### Enhancements
+
+- Installation through PyPI - Named as [OTX](https://pypi.org/project/otx/)
+- CLI update - Update `find` command to find configurations of tasks/algorithms - Introduce `build` command to customize task or model configurations - Automatic algorihm selection for the `train` command using the given input dataset
```
Comment (goodsong81) [inline CHANGELOG.md]:
```suggestion
- Enhance `find` command to find configurations of supported tasks / algorithms / models / backbones
- Introduce `build` command to customize task or model configurations in isolated workspace
- Auto-config feature to automatically select the right algorithm and default model for the `train` & `build` command by detecting the task type of given input dataset
```
Could you move last 2 items to new features?

File: CHANGELOG.md (hunk 2 above)
Comment (yunchu) [reply CHANGELOG.md]: actually those statement came from the template of the LMC (exceptional approval for releasing with vulnerable dependency) so I'm not sure whether we can change it or not. I will check it again.

Comment (yunchu) [reply CHANGELOG.md]: @goodsong81 I've added new required statement for this release.

File: CHANGELOG.md (hunk 3 above)
Comment (yunchu) [reply CHANGELOG.md]: Revised.

File: CHANGELOG.md (hunk 5 above)
Comment (yunchu) [reply CHANGELOG.md]: Revised.

**Review by goodsong81 — APPROVED** [review]
LGTM! Thanks.

**Pull request merged**